chia-tools/src/bin/analyze-chain.rs

```diff
@@ -8,7 +8,7 @@ use std::time::SystemTime;
 use sqlite::State;
 
 use chia::gen::conditions::parse_spends;
-use chia::gen::flags::MEMPOOL_MODE;
+use chia::gen::flags::{HARD_FORK, MEMPOOL_MODE};
```

Comment:
This may be a bit pedantic, but you might consider calling it `HARD_FORK_1`. Unless you think this is the last hard fork.

Reply:
As much as I know Bram wishes that we'll never have another hard fork... I remain a bit more skeptical...